examples/code-http-anthropic/n8n-kit/.bundle-py/INzyo95PUum4xEuiL9MT/generate-prompt.py

```
@@ -0,0 +1,15 @@
+            "prompt": f"Reply as if you were a gangster."
+        }
+    if(input["type"] == "type2"):
+        return {
+            "prompt": f"Reply as if you were a pirate."
+        }
+
+    raise Exception("Invalid type")
+
+return handler({
+  "type": _('webhook').item.json.query.type
+});
```

Ruff (0.12.2)

`return` statement outside of a function/method

(F706)